### ISSUE TITLE
Feature Request: Add "Copy To" and "Move To" to the right click menu

### ISSUE THREAD

**Issue by AJReissig** [elementary/files#417]
While Elementary Files does have an effective copy and paste, I believe a worthwhile addition would be to add "Copy To" and "Move To" to the right click menu, much like Nautilus Files has (see below image).  When you do a fair amount of copy/move/paste, I find this to be a huge time saver, more so than having two windows of the file manager open for dragging and dropping.

![copyto](https://user-images.githubusercontent.com/32073347/38781464-5f0ee5a6-40b3-11e8-98d4-6b133a3d4ab0.png)

**Comment by cassidyjames** [comment]
@AJReissig what happens in Nautilus when selecting those items?

**Comment by AJReissig** [comment]
You have a new window that pops open and you can select where you want the file (or files) copied or moved.  See below image.
![move_dialog](https://user-images.githubusercontent.com/32073347/38842718-872e0ab0-41b9-11e8-988f-5efc11e6f8de.png)

**Comment by jeremypw** [comment]
@danirabbit Does having a more extensive context menu fit with elementary design principles - is there a compact way of achieving this?

**Comment by danirabbit** [comment]
@jeremypw after the Gtk4 port we can probably do things to make the menu a lot more organized/compact with a popover. I'm not personally opposed to this feature if you think it makes sense. It looks like it's just opening a file chooser dialog?

**Comment by jeremypw** [comment]
Yes it looke like its just a question of opening a filechooser and performing the operation. Only question is whether to open the target folder in Files afterwards.
